uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-notification-badge.js

```diff
@@ -0,0 +1,19 @@
+(function (CRM, require) {
+  var extPath = CRM.tasksAssignments.extensionPath + 'js/src/tasks-notification-badge';
+
+  require.config({
+    urlArgs: 'bust=' + (new Date()).getTime(),
+    paths: {
+      'tasks-notification-badge': extPath
+    }
+  });
+
+  require([
+    'tasks-notification-badge/app'
+  ],
+  function (angular) {
+    angular.bootstrap(
+      document.querySelector('[data-tasks-notification-badge]'), ['tasks-notification-badge']
+    );
+  });
+})(CRM, require);
```

Comment:
why are we creating a new module just for these notifications? Why not just add the notifications inside `/js/src/tasks-assignments/components` ?

Reply:
We need a new module, because in the end we need to create a separate `.min.js` which would be imported by the SSP Menu. Reference https://github.com/compucorp/civihr-employee-portal/pull/469/files#diff-0629dcabaa6a0ce1bb40f802dbd6e24cR6574

If we dont do this, we would need to import the whole T&A module just for the small badge. 

The same was followed in L&A too, where we created a new module named `manager-notification-badge`.